occult/lib/all.dart

```diff
@@ -443,20 +443,23 @@ Future<void> createFirebaseRC(String projectname) async {
 }
 
 Future<void> enableGcloudArtifactRegistry(String project) async {
+  final String gcloudCommand = Platform.isWindows ? 'gcloud.cmd' : 'gcloud';
+
   final runner = Spinner(
     icon: '✔'.padRight(2).green(),
     leftPrompt: (done) => '',
     rightPrompt: (done) => done
         ? 'Enabled GCP Artifact Registry'
         : 'Enabling GCP Artifact Registry',
   ).interact();
-  //.// gcloud services enable run.googleapis.com artifactregistry.googleapis.com --project=YOUR_PROJECT_ID
-  ProcessResult p = await Process.run("gcloud", [
+
+  ProcessResult p = await Process.run(gcloudCommand, [
```

Comment:
This should be calling "gcloud" make a process utility that calls process.run switching "command" to "${command}.cmd" on windows etc